package.json

```diff
@@ -39,14 +39,14 @@
   },
   "scripts": {
     "build": "gulp",
-    "feature-tests": "./node_modules/.bin/cucumber-js test/feature --format=progress",
+    "feature-tests": "cucumber-js test/feature --format=progress",
```

Comment:
Any reasons for taking this out? If we leave that path in, nobody gets hurt, and it works for more people.